frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx

```diff
@@ -127,7 +133,18 @@ const Footer: React.FC<FooterProps> = ({
             isReview
               ? onFinish
               : isSelectTemplate
-              ? () => withSupportModal(template.variants[0], onNext)
+              ? () => {
```

Comment:
I know you didn't create this nested ternary, but I think it would be better to define a function above and use `if` statements to make this easier to read. It will also cause less visual clutter in the props list.